nvflare/private/fed/server/server_engine.py

```diff
@@ -149,11 +147,14 @@ def get_engine_info(self) -> EngineInfo:
 
         return self.engine_info
 
-    def get_run_info(self) -> RunInfo:
+    def get_run_info(self) -> dict:
```

Comment:
Better to return a well defined RunInfo object, instead of free dict here, even though the RunInfo does not contain too much info yet.